frontend/static/quotes/english.json

```
@@ -38805,6 +38805,24 @@
       "length": 186,
       "approvedBy": "Smithster",
       "id": 6744
+    },
+    {
+      "text": "I was fast asleep, you stood there by the bedside waiting to haunt by day, someday we might meet, I won't shed a tear for the things that you stole. Your roots go far and deep, they strangle. Frantic breathing needed to stay afloat.",
+      "source": "Quiet Commotion",
+      "id": 6745,
+      "length": 232
+    },
+    {
+    "text": "Finding myself back to earth, I never wanna come back down. I'm fine here just where I am, with a full moon in my arms.",
+    "source": "Quiet Commotion",
+    "id": 6746,
+    "length": 119
+    },
+    {
+  "text": "Snow's falling down to the ground over secrets I have hid, bleach my sweet mind with the rays of a dying purple sun. Breathing sulfur, my lungs weren't built to last too long.",
+  "source": "Quiet Commotion",
+  "id": 6747,
+  "length": 175 
```

Almost there. The body of the code blocks are also indented. Accepting this change(there is a button below the diff) will fix it.
```suggestion
    {
      "text": "I was fast asleep, you stood there by the bedside waiting to haunt by day, someday we might meet, I won't shed a tear for the things that you stole. Your roots go far and deep, they strangle. Frantic breathing needed to stay afloat.",
      "source": "Quiet Commotion",
      "id": 6745,
      "length": 232
    },
    {
      "text": "Finding myself back to earth, I never wanna come back down. I'm fine here just where I am, with a full moon in my arms.",
      "source": "Quiet Commotion",
      "id": 6746,
      "length": 119
    },
    {
      "text": "Snow's falling down to the ground over secrets I have hid, bleach my sweet mind with the rays of a dying purple sun. Breathing sulfur, my lungs weren't built to last too long.",
      "source": "Quiet Commotion",
      "id": 6747,
      "length": 175
```